File: SecondYear/Divyansh670_DivyanshSrivastav_2226it1029_2/Week_1/data_types.java
import java.util.Scanner;

public class data_types {

    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        
        String dataType = scanner.next();

        switch (dataType) {
            case "byte":
                System.out.println(Byte.SIZE/8);
                break;
            case "short":
                System.out.println(Short.SIZE/8);
                break;
            case "int":
                System.out.println(Integer.SIZE/8);
                break;
            case "long":
                System.out.println(Long.SIZE/8);
                break;
            case "float":
                System.out.println(Float.SIZE/8);
                break;
            case "double":
                System.out.println(Double.SIZE/8);
                break;
            case "char":
                System.out.println(Character.SIZE/8);
                break;
            default:
                System.out.println("Invalid data type.");
        }
    }
}

File: SecondYear/aayushker_AayushkerSingh_2226CSE(AI)1010_2/Week_1/Data_types.py
def dataTypes(data_type: str):
    if data_type == "Integer":
        return 4
    elif data_type == "Long":
        return 8
    elif data_type == "Double":
        return 8
    elif data_type == "Float":
        return 4
    elif data_type == "Character":
        return 1
    else:
        print("Unknown data type")